tools/tune/eq/example_spk_eq.m

```diff
@@ -125,7 +125,7 @@ function example_spk_eq()
         bm_fir = eq_fir_blob_merge(channels_in_config, ...
                 num_responses, ...
                 assign_response, ...
-                [ bq_fir ]);
+                ( bq_fir ));
```

Comment:
Why [] to (), did you check if it impacts the exported blob?

Reply:
To specify expression precedence , I 've to use parentheses () rather than brackets [].
I did the checked the impact  with `()` and `[]`, i don't see any  visible change in data. On the safe side, I remove both braces.